test/hotspot/jtreg/runtime/cds/appcds/dynamicArchive/LambdaForClassInBaseArchive.java

```diff
@@ -75,7 +75,7 @@ private static void doTestCustomBase(String baseArchiveName, String topArchiveNa
               appClass, "lambda")
             .assertNormalExit(out -> {
                     out.shouldHaveExitValue(0)
-                       .shouldContain("Archiving hidden SimpleApp$$Lambda$1");
+                       .shouldMatch("Archiving hidden SimpleApp[$][$]Lambda[$][\\d+]*");
```

Comment:
That a zero-or-more digit match or a one-or-more digit match?
I think the pattern should require 1 or more digits.

Reply:
The `\\d+` means one or more digits.

Reply:
Thanks for clarifying (for my tired brain).